cmd/zfs/zfs_main.c

```diff
@@ -4721,8 +4817,11 @@ zfs_do_send(int argc, char **argv)
 	if (flags.replicate && fromname == NULL)
 		flags.doall = B_TRUE;
 
-	err = zfs_send(zhp, fromname, toname, &flags, STDOUT_FILENO, NULL, 0,
-	    flags.verbosity >= 3 ? &dbgnv : NULL);
+	err = zfs_send(zhp, fromname, toname, &flags, STDOUT_FILENO,
+	    exclude_context.count > 0 ? zfs_do_send_exclude : NULL,
```

Comment:
It is interesting that this parameter exists even though we don't use it anywhere in the codebase as far as I can tell. I wonder why?

Reply:
I know! I thought that too! I assume there was some use for it at some point in history?